Devscord.DiscordFramework.UnitTests/Commands/RunnerOfIBotCommandMethodsTests/Controllers/TestController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using Devscord.DiscordFramework.UnitTests.Commands.RunnerOfIBotCommandMethodsTests.BotCommands;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+
+namespace Devscord.DiscordFramework.UnitTests.Commands.RunnerOfIBotCommandMethodsTests.Controllers
+{
+    class TestController : IController
```

Comment:
https://pl.wikipedia.org/wiki/Test_jednostkowy

Reply:
o co ci chodzi? Jak bez kontrollera mogę wykonać testy?

Reply:
1 poczytaj jaki jest cel testów jednostkowych
2 poczytaj o mockowaniu
3 zastanów sie którą pojedynczą funkcjonalność chcesz przetestować i czy jest w ogóle sens ją testować

Reply:
czyli mam rozumieć, że mam zrobić mocka do IController, bo jeśli będę używał TestController to wyjdzie na to, że oprócz testować komendy, będę też testował przez przypadek tego kontrolera?

Reply:
tak